src/librustdoc/html/render.rs

```diff
@@ -1291,6 +1291,7 @@ fn settings(root_path: &str, suffix: &str) -> String {
         ("auto-hide-method-docs", "Auto-hide item methods' documentation", false).into(),
         ("auto-hide-trait-implementations", "Auto-hide trait implementations documentation", true)
             .into(),
+        ("auto-collapse-implementors", "Auto-collapse implementors", true).into(),
```

Comment:
I think this is confusing, I wonder if we can just merge this with the previous option?

Or perhaps two settings:

 - Auto-hide trait implementation documentation
 - Auto-hide trait implementation items

Reply:
Hum... Maybe a rewording? But I think it'd be nice trying to make settings more obvious more generally. In this case, we target just the implementors (the section and its children), not the trait implementations generally.

Reply:
Yeah I just think that when you have so many it gets confusing. Definitely when I look at the page I'm unsure as to what some of the options do.

Rewording seems fine

Reply:
I'm open to suggestions then! Btw: should I do it in this PR directly or should I open a new one?

Reply:
Do it here itself. I suggested options for the wording above for the two settings

Reply:
Oh indeed! Thanks, sending an update shortly.